agenta-backend/agenta_backend/services/docker_utils.py

```diff
@@ -90,7 +92,7 @@ def start_container(image_name, app_name, variant_name, env_vars: DockerEnvVars)
             detach=True,
             labels=labels,
             network="agenta-network",
-            name=f"{app_name}-{variant_name}",
+            name=f"{app_name}-{variant_name}-{user_id}",
```

Comment:
I think we also need to change the line 105 (URI) and the logic for that! The URI should become /user_id/app_name/variant_name/ otherwise if two users have the same appname and variant name we'll reverse proxy them to the same link

Reply:
Note that the logic for the URI is hardcoded in many places (i.e. in the frontend, maybe CLI)

Reply:
Line 108 expects the container name to be app_name-variaant_name. That also needs to change

Reply:
Alright, thank you.